Automattic/simplenote-android
# Update Pinned Icon

### Fix
Update the pinned icon (formerly named radio checked icon) in the note list.  See the screenshots below for illustration.

![update_pinned_icon](https://user-images.githubusercontent.com/3827611/66054613-84eebb00-e4f1-11e9-8549-9e98f809001f.png)

### Test
1. Notice pinned icon for pinned notes in note list.
2. Long-press any note in list.
3. Notice pinned icon for ***Pin to top*** action in app bar.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
